app/tests/components/Review/ReviewTheme.ts

```diff
@@ -0,0 +1,430 @@
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('Fibre,Fixed wireless');
+  });
+
+  it('should have correct fields in The Project planning section', () => {
+    renderTest(mockQueryPayloadWithFormData);
+
+    const section = within(
+      screen
+        .getByRole('heading', { name: /Project planning/i })
+        .closest('section')
+    );
+
+    expect(
+      section
+        .getByText('Project Start Date (YYYY/MM/DD)')
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('2022-06-10');
+  });
+
+  it('should have correct fields in The Other funding sources section', () => {
+    renderTest(mockQueryPayloadWithFormData);
+
+    const section = within(
+      screen
+        .getByRole('heading', { name: 'Other funding sources' })
+        .closest('section')
+    );
+
+    expect(
+      section
+        .getAllByText('2022-23')[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$1');
+
+    expect(
+      section
+        .getAllByText('2023-24')[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$2');
+
+    expect(
+      section
+        .getAllByText('2024-25')[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$3');
+
+    expect(
+      section
+        .getAllByText('2025-26')[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$4');
+
+    expect(
+      section
+        .getAllByText('2026-27')[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$5');
+
+    expect(
+      section
+        .getByText(/Total amount requested under Canadian Infrastructure Bank/i)
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent('$15');
+  });
+
+  it('should the correct fields in the Organization location section', () => {
+    renderTest(mockQueryPayloadWithFormData);
+
+    const section = within(
+      screen
+        .getByRole('heading', { name: 'Organization location' })
+        .closest('section')
+    );
+
+    expect(
+      section
+        .getAllByText(/city/i)[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent(/victoria/i);
+
+    expect(
+      section
+        .getAllByText(/unit number/i)[0]
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent(/1231231/i);
+
+    expect(
+      section
+        .getByText(/Is the mailing address the same as above/i)
+        .closest('tr')
+        .getElementsByTagName('td')[0]
+    ).toHaveTextContent(/no/i);
+  });
+
+  it('should have the correct field and value in the Organization profile section', () => {
+    renderTest(mockQueryPayloadWithFormData);
+
+    expect(document.getElementById('bandNumber-error')).toBeNull();
+  });
+};
+
+// eslint-disable-next-line jest/no-export
+export default sharedReviewThemeTests;
```

Comment:
@matthieu-foucault Not sure what to do about this. Not that exporting from tests is something we will do regularly but I'm not sure I see the benefit to this rule.

Reply:
Figuring out why our eslint check ignores the `// eslint-disable-next-line` comments would be great.
The rule can be useful to avoid running tests multiple times, but it's a false positive here.

Reply:
You want me to debug that as part of this ticket?

Reply:
A separate PR would be better, to avoid adding more noise to this one